download/index.markdown

```diff
@@ -7,111 +7,149 @@ categories: [ _nav, _6 ]
+### Debian
 
 `apt-get install irssi`.  
 You may find more up to date version on [Debian Backports][5]{:rel='external'}
 
-[3]: /assets/debian.png
-[4]: /assets/ubuntu.png
 [5]: http://backports.debian.org/
 
 </div>
-<div class="col-md-4" markdown="1">
+<div class="info about-ubuntu" markdown="1">
+
+### Ubuntu
+
+`apt-get install irssi`.  
+
+</div>
+<div class="info about-macosx" markdown="1">
 
-![][6]{:class='osicon'}**OS X**
+### OS X
 
 Install [Homebrew][7]{:rel='external'}, then  
 `brew install irssi`  
 You can also install Git with `--HEAD`
 
-[6]: /assets/macosx.png
 [7]: http://brew.sh/
 
 </div>
-</div>
-<div class="row">
-<div class="col-md-4" markdown="1">
+<div class="info about-gentoo" markdown="1">
 
-![][8]{:class='osicon'}**Gentoo**
+### Gentoo
 
 `emerge irssi`
 
-[8]: /assets/gentoo.png
-
 </div>
-<div class="col-md-4" markdown="1">
+<div class="info about-openSUSE" markdown="1">
 
-![][19]{:class='osicon'}**SuSE**
+### SuSE
 
 `zypper in irssi`.
 
 You may find more up to date versions on [openSUSE Package Search][20]{:rel='external'}
 
-[19]: /assets/openSUSE.png
 [20]: http://software.opensuse.org/package/irssi
 
 </div>
-<div class="col-md-4" markdown="1">
+<div class="info about-arch" markdown="1">
 
-![][9]{:class='osicon'}**ArchLinux**
+### ArchLinux
 
 `pacman -S irssi`
 
-[9]: /assets/arch.png
-
-</div>
 </div>
-<div class="row">
-<div class="col-md-4" markdown="1">
+<div class="info about-fedora" markdown="1">
 
-![][10]{:class='osicon'}**Fedora/Red Hat**
+### Fedora
 
 `dnf install irssi` or  
 `yum install irssi`
 
-[10]: /assets/fedora.png
-
 </div>
-<div class="col-md-4" markdown="1">
+<div class="info about-opencsw" markdown="1">
 
-![][11]{:class='osicon'}**Solaris**
+### Solaris
 
 `pkg-get install irssi`
 
-[11]: /assets/opencsw.png
-
 </div>
-<div class="col-md-4" markdown="1">
+<div class="info about-slackware" markdown="1">
 
-![][12]{:class='osicon'}**Slackware**
+### Slackware
 
 `slackpkg install irssi`
 
-[12]: /assets/slackware.png
+</div>
+
+<!-- unfortunately we have to hard-code the display criteria for obs-info in the style.css -->
+<div class="row obs-info" markdown="1">
+<div class="col-sm-2 col-md-3" markdown="1">
+
+![][19]{:class='obsicon'}
 
 </div>
+<div class="col-sm-10 col-md-9">
+<div class="col-sm-12 col-lg-6" markdown="1">
+
+The [openSUSE Build Service](http://build.opensuse.org/){:rel='external'} is building Irssi release packages and Irssi git snapshot packages for Debian, Fedora, Ubuntu, and openSUSE.
+
 </div>
+<div class="irssi-git col-sm-6" markdown="1">
 
-## Sources
+[Install irssi-git](//software.opensuse.org/download.html?project=home:ailin_nemui:irssi-git;package=irssi-git){:rel='external'}{:class='btn btn-info'}.
+
+</div>
+<div class="irssi-release col-sm-6" markdown="1">
+
+[Install irssi release](//software.opensuse.org/download.html?project=home:ailin_nemui:irssi-test;package=irssi){:rel='external'}{:class='btn btn-success'}.
```

Comment:
Maybe remove {:rel='external'} here and in the previous button? Makes it look like it doesn't have padding-right

![](http://dump.dequis.org/fdzbY.png)

Also nuke that dot at the end.